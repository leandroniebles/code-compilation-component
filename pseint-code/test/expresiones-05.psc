Proceso sin_titulo
	A<-"A";
	Escribir "a" 2;
	Escribir 2 "a";
	Escribir A "a";
	Escribir "a" A;
	Escribir (1) A;
	Escribir A (1);
FinProceso
