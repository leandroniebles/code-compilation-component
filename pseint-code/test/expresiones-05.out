a2
2a
Aa
aA
1A
A1
